compatibility-verifier/control-service.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# A helper script that starts/stops given component for a given build of Pinot
+# This script is intended to be called stand-alone or from comp-verifier.sh 
+# command syntax: ./control-service.sh command component target_dir pinot_version
+
+# verify correct usage of this script 
+if [[ $# -ne 4 ]]; then
+  printf "You used %s arguments \n" "$#"
+  printf "Usage: \n ./control-service.sh command component target_dir pinot_version \n"
+  exit 1
+fi
+
+# get arguments
+command=$1
+component=$2
+target_dir=$3
+pinot_version=$4
+
+declare -a commands=("start" "stop")
+declare -a components=("controller" "broker" "server" "zookeeper")
+
+if [[ ! -d "$target_dir"/incubator-pinot ]]; then
+  printf "%s/incubator-pinot does not exist \n " "$target_dir"
+  exit 1
+fi
+
+# Handle invalid arguments
+if [[ ! " ${commands[*]} " =~ $command  ]]; then
+  printf "%s is not a valid command. Command must be one of: start and stop\n" "$command"
+  printf "Usage: \n ./controlService.sh command component target_dir \n"
+  exit 1
+fi
+if [[ ! " ${components[*]} " =~ $component  ]]; then
+  printf "Not a valid component. Needs to be one of: %s \n" "${components[*]}"
+  exit 1
+fi
+
+# Define ports for the test
+zkPort=2181
```

Comment:
For now, just use the default ports on the service. Let us get a basic framework working. and then we can add the bells and whistles.